gridcomps/History3G/HistoryCollectionGridComp_private.F90

```diff
@@ -215,4 +178,119 @@ function set_start_stop_time(clock, hconfig, rc) result(start_stop_time)
       _RETURN(_SUCCESS)
    end function set_start_stop_time
 
+   subroutine parse_item_expression(item, item_name, var_names, rc)
+      type(ESMF_HConfigIter), intent(in) :: item 
+      character(len=:), allocatable, intent(out) :: item_name
+      type(StringVector), intent(out) :: var_names
+      integer, optional, intent(out) :: rc
+      character(len=:), allocatable :: expression
+      integer :: status
+
+      call parse_item_common(item, item_name, expression, _RC)
+      var_names = get_expression_variables(expression, _RC) 
+
+      _RETURN(_SUCCESS)
+   end subroutine parse_item_expression
+
+   subroutine parse_item_simple(item, item_name, var_name, rc)
```

Comment:
Is `parse_item_simple` even needed or used anymore? Seems like only the expression version that returns a StringVector is used.

Reply:
`create_output_bundle` uses `parse_item_simple`. It could be modified to call:

> `call parse_item(iter, alias, short_names, _RC)`
> `call ESMF_StateGet(import_state, short_names%front(), field, _RC)`

Reply:
oh, ok, missed that, that's fine then